backend/game/irepository/test_irepository_dice.py

```diff
@@ -1,86 +1,86 @@
-                                     DieValue.TWO, DieValue.THREE, 0, 0, 1) == 1
-
-
-@pytest.mark.django_db(transaction=True)
-def test_get_dice_db():
-    repository_dice = IRepositoryDice()
-    repository_game = IRepositoryGame()
-    repository_game.save_game('Room2')
-    repository_player = IRepositoryPlayer()
-    player = Player()
-    player.set_monster_name('Godzilla1')
-    player.set_username('Test_Get')
-    repository_player.save_player(player, 'Room2')
-    dice = DiceHandler()
-    repository_dice.save_dice(player, 'Room2', DieValue.ONE, DieValue.TWO, DieValue.THREE, DieValue.ONE,
-                              DieValue.TWO, DieValue.THREE, 0, 0, 1)
-    assert repository_dice.get_dice_by_id(2).dice2 == 'DieValue.TWO'
-
-
-@pytest.mark.django_db(transaction=True)
-def test_update_dice_db():
-    repository_dice = IRepositoryDice()
-    repository_game = IRepositoryGame()
-    repository_game.save_game('Room3')
-    repository_player = IRepositoryPlayer()
-    player = Player()
-    player.set_monster_name('Godzilla2')
-    player.set_username('Test_Update1')
-    repository_player.save_player(player, 'Room3')
-    repository_dice.save_dice(player, 'Room3', DieValue.ONE, DieValue.TWO, DieValue.THREE, DieValue.ONE,
-                              DieValue.TWO, DieValue.THREE, 0, 0, 1)
-    repository_dice.update_dice_by_id(3, DieValue.THREE, DieValue.ONE, DieValue.TWO, DieValue.THREE,
-                                      DieValue.ONE, DieValue.TWO, 0, 0, 1)
-    assert repository_dice.get_dice_by_id(3).dice2 == 'DieValue.ONE'
-
-
-@pytest.mark.django_db(transaction=True)
-def test_update_die_by_id():
-    repository_dice = IRepositoryDice()
-    repository_game = IRepositoryGame()
-    repository_game.save_game('Room4')
-    repository_player = IRepositoryPlayer()
-    player = Player()
-    player.set_monster_name('Godzilla2')
-    player.set_username('Test_Update1')
-    repository_player.save_player(player, 'Room4')
-    repository_dice.save_dice(player, 'Room4', DieValue.ONE, DieValue.TWO, DieValue.THREE, DieValue.ONE,
-                              DieValue.TWO, DieValue.THREE, 0, 0, 1)
-    repository_dice.update_die_by_id(4, 2, DieValue.THREE, 1)
-    assert repository_dice.get_dice_by_id(4).dice2 == 'DieValue.THREE'
-
-
-@pytest.mark.django_db(transaction=True)
-def test_delete_dice_by_id():
-    repository_dice = IRepositoryDice()
-    repository_game = IRepositoryGame()
-    repository_game.save_game('Room5')
-    repository_player = IRepositoryPlayer()
-    player = Player()
-    player.set_monster_name('Godzilla1')
-    player.set_username('Test_Get')
-    repository_player.save_player(player, 'Room5')
-    repository_dice.save_dice(player, 'Room5', DieValue.ONE, DieValue.TWO, DieValue.THREE, DieValue.ONE,
-                              DieValue.TWO, DieValue.THREE, 0, 0, 1)
-    assert repository_dice.delete_dice_by_id(5) is None
+# import pytest
```

Comment:
Is this all just commented out to allow pytest to work on our local machines? If so I don't think you need to do that. I would argue to leave the code in, and let people know they should run ` py.test backend --ignore backend/game/db --ignore backend/game/irepository` so this doesn't look like  unnecessary code. Others may disagree with me though

Reply:
Agree, though it kind if stinks we need remember to did this.

Reply:
Agree, though it kind if stinks we need remember to did this.

Reply:
true did that because keith requested earlier, will add back as you both agree to do so

Reply:
I added a line if you are using VS code that will automatically remove these tests; however, if using PyCharm or another software unsure if it will be excluded automatically

Reply:
Great and works for me, will merge once approved

Will need one approval to move on to merge